src/components/splash.jsx

```diff
@@ -45,10 +47,53 @@ export default function Splash({
     </form>
   );
 
+  const learnMore = (
+    <Collapsible trigger="Learn more">
+      <h3>General</h3>
+      <p>Make sure that you have the right privileges to list apps or view the app.</p>
+      <h3>Engine running in Docker container</h3>
+      <p>
+        For connecting to an Engine running in a Docker container with a mounted data volume containing the app and starting
```

Comment:
Long sentence. Consider adding a comma. E.g.  
For connecting to an Engine running in a docker container, with a mounted...